### PR TITLE
add ClickHouseTooMuchMutations alert and DO_NOT_CHOWN=1 example

### PR DESCRIPTION
**Pull request by Slach**
- add ClickHouseTooMuchMutations alert
- add chi_clickhouse_table_mutations chi_clickhouse_table_mutations_parts_to_do metrics to Grafana Dashboard
- add securityContext example without chown -R with clickhouse-server:latest image

Signed-off-by: Eugene Klimov <eklimov@altinity.com>

### CODE REVIEW

File: grafana-dashboard/Altinity_ClickHouse_Operator_dashboard.json [hunk 1]
```diff
@@ -3332,7 +3332,7 @@
       "dashLength": 10,
       "dashes": false,
       "datasource": "${DS_PROMETHEUS}",
-      "description": "Number of mutations (ALTER DELETE/ALTER UPDATE)",
+      "description": "Number of mutations (ALTER DELETE/ALTER UPDATE) which not complete and how much parts should be mutate",
```
Comment (alex-zaitsev) [inline grafana-dashboard/Altinity_ClickHouse_Operator_dashboard.json:3335]: Please name it 'Number of active mutations (ALTER TABLE UPDATE/DELETE)'.

File: grafana-dashboard/Altinity_ClickHouse_Operator_dashboard.json [hunk 2]
```diff
@@ -3372,12 +3382,17 @@
...
       "spaceLength": 10,
       "stack": false,
-      "steppedLine": false,
+      "steppedLine": true,
       "targets": [
         {
-          "expr": "chi_clickhouse_metric_PartMutation{exported_namespace=~\"$exported_namespace\",chi=~\"$chi\",hostname=~\"$hostname\"}",
-          "legendFormat": "{{hostname}}",
+          "expr": "chi_clickhouse_table_mutations{exported_namespace=~\"$exported_namespace\",chi=~\"$chi\",hostname=~\"$hostname\"}",
+          "legendFormat": "mutations {{database}} {{table}} {{hostname}}",
```
Comment (alex-zaitsev) [inline grafana-dashboard/Altinity_ClickHouse_Operator_dashboard.json:3389]: Is it going to display different series for different databases/tables?

Comment (Slach) [reply grafana-dashboard/Altinity_ClickHouse_Operator_dashboard.json:3389]: yes 
do you want reduce it to hostname only?

Comment (alex-zaitsev) [reply grafana-dashboard/Altinity_ClickHouse_Operator_dashboard.json:3389]: Yes, let's make it hostname level. Users may go to mutations table for the detail

Comment (Slach) [reply grafana-dashboard/Altinity_ClickHouse_Operator_dashboard.json]: fixed in https://github.com/Altinity/clickhouse-operator/pull/659/commits/8added93490bff83fce7d9a37ff0fef64189d36e

File: grafana-dashboard/Altinity_ClickHouse_Operator_dashboard.json [hunk 3]
```diff
@@ -3332,7 +3332,7 @@
       "dashLength": 10,
       "dashes": false,
       "datasource": "${DS_PROMETHEUS}",
-      "description": "Number of mutations (ALTER DELETE/ALTER UPDATE)",
+      "description": "Number of active mutations (ALTER DELETE/ALTER UPDATE) and how much parts should be mutate",
```
Comment (alex-zaitsev) [inline grafana-dashboard/Altinity_ClickHouse_Operator_dashboard.json:3335]: Let's make it shorter:

"Number of active mutations (ALTER DELETE/ALTER UPDATE) and parts to mutate"

File: grafana-dashboard/Altinity_ClickHouse_Operator_dashboard.json (hunk 1 above)
Comment (Slach) [reply grafana-dashboard/Altinity_ClickHouse_Operator_dashboard.json]: fixed in https://github.com/Altinity/infra/commit/59e09a75f9bc42ee21987aec61d0d0af8bc5feaf

**Pull request merged**